mesonbuild/interpreter/interpreter.py

```diff
@@ -2410,7 +2410,8 @@ def _warn_about_builtin_args(self, args: T.List[str]) -> None:
             elif arg == '-g':
                 mlog.warning(f'Consider using the built-in debug option instead of using "{arg}".',
                              location=self.current_node)
-            elif arg.startswith('-fsanitize'):
+            elif (arg.startswith('-fsanitize=') or arg.startswith('/fsanitize=')) and \
+                 set(arg[11:].split(',')) <= {'none', 'address', 'thread', 'memory', 'undefined'}:
```

Comment:
Did you mean `arg[11].split(',')`?

Also, you writing `not set(arg[11].split(',')).issubset(...)`?

Reply:
`arg[11:]` is everything after `-fsanitize=` or `/fsanitize=`.

`<=` is the same as `issubset`, so it warns if the user's specified values are all supported by `-Db_sanitize=`.